### ISSUE TITLE
Support for Edit

### ISSUE THREAD

**Issue by alexvonme** [ChrisChinchilla/vale-vscode#45]
Does the extension support [the action to](https://vale.sh/docs/topics/actions/#edit) ``edit`` from Vale?

I've tried [the example](https://github.com/errata-ai/vale/issues/851#issuecomment-2179442444) given by the lead developer of Vale, but it doesn't trigger the appropriate fix. The ``Quick Fix`` option just gives out the ``No suggestion found`` message.

I thought this might be a problem with the extension because the screenshot from the developer (which I presume is from Sublime Test) seems to suggest the correct answer yet I haven't been able to get the same ``Quick Fix`` option in VSCode. I also tried the other parameters from ``edit`` such as ``trim_right`` and realized I can't get the correct ``Quick Fix`` options for them either. Only ``No suggestion found``.

The extension supports the ``replace`` and ``suggest`` actions, so I was hoping that it could also support, the admittedly more complicated, ``edit``.

**Comment by jdkato** [comment]
This will be possible once #23 is complete.

**Comment by ChrisChinchilla** [comment]
Yes @alexvonme What @jdkato says is mostly true, I think the extension supported (or supports) some `edit` actions, but it depends on the style rule as the extension then has to add support for every potential option. So I paused development of those in favour of attempting to work on #23 instead, but that’s taking some time due to all sorts of complications (and work 😅). Getting back to it very soon though.

**Comment by alexvonme** [comment]
@ChrisChinchilla thank you for the detailed answer. I asked in part to confirm that it wasn't an issue specific to my configuration. We'll be tracking the pull request as well.

**Comment by ChrisChinchilla** [comment]
@alexvonme and @ccoVeille As far as I know, this is now supported, if you have any specific cases, let me know.
